chain-spec-generator/src/common.rs

```diff
@@ -90,6 +90,8 @@ pub fn from_json_file(filepath: &str, supported: String) -> Result<Box<dyn Chain
 			Ok(Box::new(GluttonKusamaChainSpec::from_json_file(path)?)),
 		x if x.starts_with("encointer-kusama") =>
 			Ok(Box::new(EncointerKusamaChainSpec::from_json_file(path)?)),
+		x if x.starts_with("people-kusama") =>
```

Comment:
Hmm, I just wonder why coretime-kusama is also missing, could you add it also?

Reply:
I think we could extract these IFs for chain-id matching out of the common.rs (it is not really common) to the main.rs and e.g. join it with supported networks cfg here: https://github.com/polkadot-fellows/runtimes/blob/main/chain-spec-generator/src/main.rs#L39